### PR TITLE
Burak-news

### PR DESCRIPTION
**Pull request by myniqx**
https://github.com/freedomcombination/monorepo/issues/780

### CODE REVIEW

File: apps/trend-rights/pages/api/images.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { imagesRouter } from '@fc/services/src/api/images'
```
Comment (7alip) [inline apps/trend-rights/pages/api/images.ts:1]: Is this API route being used?

**Review by 7alip — COMMENTED** [review]
image

I discussed with @MustafaTRKYLMZ and it's probably cleaner not to combine topics in one page but show blogs and activities in different tabs. They will still be displayed in Topic card the same way you implemented.

You can also use only client side fetch and add skeletons while fetching. I would create custom hooks for it.

An example approach

```tsx
const useMappedBlogs = () => {
  const { data, ...blogsQuery } = useStrapiRequest<Blog>()
  
  const mappedBlogTopics = data?.data?.map()
  
  return { data: mappedBlogTopics, ...blogsQuery }
}
```

Each TabContent can its own component. Since you use localStorage in Activity and Blog components, they can be imported dynamically in the client.

Chakra tab contents are loaded dynamically. If you fetch the content inside the tab content components, the fetch will be trigger only the tab is active.

File: apps/trend-rights/pages/news.tsx [hunk 2]
```diff
@@ -152,24 +177,142 @@ export const getServerSideProps = async (
...
-  }))
+
+  const topics = queryClient.getQueryData<TopicBase[]>(['topics']) || []
 
   return {
     props: {
       seo,
       topic: recommendedTopic as RecommendedTopic | null,
-      topics: recommendedTopics,
+      topics,
```
Comment (7alip) [inline apps/trend-rights/pages/news.tsx:191]: I noticed that we don't use this page for SEO purposes. When we share a news card, we point to its original source.

For blogs and activities, we can point to their original pages in foundation website. In this case we don't have to do server side fetch and `topic` is redundant here.

**Comment by myniqx** [comment]
this is final state:
image

File: apps/trend-rights/pages/api/images.ts (hunk 1 above)
Comment (7alip) [reply apps/trend-rights/pages/api/images.ts:1]: @myniqx I'm wondering where this API and world-map.svg are used?

Comment (myniqx) [reply apps/trend-rights/pages/api/images.ts:1]: TopicCard -> backgroundImage
image

Comment (myniqx) [reply apps/trend-rights/pages/api/images.ts:1]: and API, when user clicks to create post ... it leads to crush. that is when i realize this.
CreateModelForm -> 
image
and useFileFormUrl ->
image

Comment (7alip) [reply apps/trend-rights/pages/api/images.ts:1]: Thanks, I just forgot what I did in the past :)

File: packages/ui/src/admin/TopicCard/TopicCard.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import {
   Popover,
   PopoverArrow,
   PopoverBody,
+  Highlight,
```
Comment (7alip) [inline packages/ui/src/admin/TopicCard/TopicCard.tsx:11]: Great! I didn't know that :)

**Review by 7alip — APPROVED** [review]
Great job! Thanks

**Pull request merged**